bin/openstackbmc

```diff
@@ -70,18 +70,12 @@ class OpenStackBmc(bmc.Bmc):
                 sys.exit(1)
 
     def get_boot_device(self):
-        server = self.novaclient.servers.get(self.instance)
-        retval = 'network' if server.metadata.get('libvirt:pxe-first') else 'hd'
+        retval = 'network'
         self.log('Reporting boot device', retval)
         return retval
 
     def set_boot_device(self, bootdevice):
-        server = self.novaclient.servers.get(self.instance)
```

Comment:
I would prefer not to remove this functionality from the bmc.  If the code is being run on an unpatched cloud then it will be ignored anyway, and for developer setups I still prefer the Nova patch for pxe booting since it means I don't have to remember to rebuild the instances every time I deploy.